fourth review/swagger.yaml

```diff
@@ -0,0 +1,1366 @@
+            properties:
+              errors:
+                  $ref: '#/definitions/Error'  
+        
+            
+            
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -X POST -H "Content-Type: application/json" https://#{api_key}:#{api_password}@your_store_domain.myshopify.com/admin/draft_orders.json -d '{
+            	"draft_order": {
+            		"email": "paul.norman@example.com",
+              		"customer": {
+                		"email": "paul.norman@example.com",
+                		"first_name": "Paul",
+                		"last_name": "Norman"
+              		},
+                	"billing_address": {
+                  		"first_name": "John",
+                  		"last_name": "Smith",
+                  		"address1": "123 Fake Street",
+                  		"phone": "555-555-5555",
+                  		"city": "Fakecity",
+                  		"province": "Ontario",
+                  		"country": "Canada",
+                  		"zip": "K2P 1L4"
+                	},
+                	"shipping_address": {
+                  		"first_name": "Jane",
+                  		"last_name": "Smith",
+                  		"address1": "123 Fake Street",
+                  		"phone": "777-777-7777",
+                  		"city": "Fakecity",
+                  		"province": "Ontario",
+                  		"country": "Canada",
+                  		"zip": "K2P 1L4"
+            		},
+              		"line_items": [
+            			{
+                  			"custom": true,
+                  			"title": "Basic White Tee",
+                  			"price": 19.99,
+                  			"quantity": 2
+            			}
+              		]
+            	}
+            }'
+            
+        - lang: Ruby
+          source: |
+              require 'shopify_api'
+
+              shop_url = "https://#{api_key}:#{api_password}@your_store_domain.myshopify.com/admin"
+              ShopifyAPI::Base.site = shop_url
+              ShopifyAPI::DraftOrder.create({
+                email: "paul.norman@example.com",
+                customer: {
+                    email: "paul.norman@example.com",
+                    first_name: "Paul",
+                    last_name: "Norman"
+                },
+                shipping_address: {
+                    first_name: "Jane",
+                    last_name: "Smith",
+                    address1: "123 Fake Street",
+                    phone: "777-777-7777",
+                    city: "Fakecity",
+                    province: "Ontario",
+                    country: "Canada",
+                    zip: "K2P 1L4"
+                },
+                line_items: [{
+                    custom: true,
+                    title: "Basic White Tee",
+                    price: 19.99,
+                    quantity: 2
+                }]
+              })    
+   
+            
+  '/admin/draft_orders/#{id}.json':
+    get:
+      description: Receive a single draft order
```

Comment:
* add end punctuation (a **.** in this case)
* down the road, we will come up with a pattern to prevent repetition across the heading and description content:

![screen shot 2016-11-28 at 1 52 59 pm](https://cloud.githubusercontent.com/assets/10617892/20681526/fef75252-b571-11e6-9b37-1e696fb1031e.png)